src/components/Chips/Chips.tsx

```diff
@@ -129,86 +129,96 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
       [onClick]
     );
 
+    const overrideClassName = cx(styles.chips, "chips--wrapper", className, {
+      [styles.disabled]: disabled,
+      [styles.withClose]: hasCloseButton,
+      [styles.noAnimation]: noAnimation,
+      [styles.withUserSelect]: allowTextSelection
+    });
+
+    const clickableProps = useClickableProps(
+      {
+        onClick: onClickCallback,
+        onMouseDown,
+        className: overrideClassName,
```

Comment:
not need to be part of hook because hook is logical only and indifferent to style settings

Reply:
Ok, let's imagine I'm removing all styles related stuff from the hook. How then it would look here
We still need to get clickable-styles from somewhere, e.g. with classNames it would look this:
```
className: cx(CLICKABLE_CSS_BASE_CLASS, overrideClassName, {
      disabled,
      [bemHelper({ state: "disable-text-selection" })]: !enableTextSelection
    })
```
So on every use of `useClickableProps` I'll need also to import `Clickable.scss` and duplicate this code? For now it would be 2 duplications - in `Clickable` and in `Chips`... Shouldn't work like this

So I can understand that hooks should contain logic and not including stylesheets, but not in this case IMHO - hook is just returning props, which you can spread on element and it would become clickable and styles related stuff are also an important part of this transformation

Reply:
let's talk over it on zoom ?

Reply:
According to what I know, hooks should be only logical and not include any styles or related. As far as I understand, useClickable is a logical hook that should receive only the relevant props for making it accessible (keyboard navigation aria...)
In other open-source implementations for similar use cases as React Aria, they also ignore styles or class names for the same reason.
I understand that this way, the developer which using the use clickable will need to implement the style by himself (disabled cursor and blocking selection of text), but this is the reason we most of the time will recommend use this clickable component over the hook and recommend using the hook only when there is no other choice.

Reply:
removed from the hook